Gemfile

```
@@ -67,6 +67,9 @@ group :development do
   # Use console on exceptions pages [https://github.com/rails/web-console]
   gem 'web-console'
 
+  # for get letter in browser
+  gem 'letter_opener', '~> 1.8'
```

Why not just
```suggestion
  gem 'letter_opener'
```
? :)